From: Christian Hofstaedtler <zeha@debian.org>
Subject: drop version from activesupport dependency

Upstream changelog noted that the explicit dependency on 3.2.x is
only there to keep running on Ruby <= 1.9.2, but as jessie only
ships 1.9.3+, we can ignore that. And Rails 3.2.x went out, so we
MUST ignore that.

Index: jekyll/jekyll.gemspec
===================================================================
--- jekyll.orig/jekyll.gemspec	2014-11-09 16:16:40.538191014 +0100
+++ jekyll/jekyll.gemspec	2014-11-09 16:17:11.414172834 +0100
@@ -56,7 +56,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency('maruku', "~> 0.7.0")
   s.add_development_dependency('rdiscount', "~> 1.6")
   s.add_development_dependency('mime-types', "~> 1.5")
-  s.add_development_dependency('activesupport', '~> 3.2.13')
+  s.add_development_dependency('activesupport')
   s.add_development_dependency('jekyll_test_plugin')
   s.add_development_dependency('jekyll_test_plugin_malicious')
   s.add_development_dependency('rouge', '~> 1.3')
